docs/theory/statistics.rst

```diff
@@ -1,10 +1,174 @@
+the equation above are zero.
+
+For a given set of parameters, enzyme concentrations and initial metabolite
+conditions, there should be a single steady state balanced metabolite
+concentration and set of fluxes.
+
+The kinetic model's role in Maud's statistical model is to connect latent
+parameters - i.e. :math:`\theta` above - with measureable quantities,
+i.e. :math:`\mathbf{m}` and :math:`\mathbf{v}`. 
+
+
+Experimental Information
+========================
+
+Maud represents information from experiments that measure enzyme concentrations
+and metabolite concentrations using the following regression model, where
+:math:`y` is the observation and :math:`\hat{y}` is the unobserved true value
+of the experimentally measured quantity:
+
+.. math::
+
+   y \sim LogNormal(\log(\hat{y}), \sigma)
+
+Measurements of reaction fluxes use the following similar regression model:
+
+.. math::
+
+   y \sim Normal(\hat{y}, \sigma)
+   
+
+In all three cases it is assumed that for each kind of measurement the error
+standard deviation :math:`\sigma` is known (though this number is in general
+different for each measured quantity in each experiment). It is the user's
+responsibility to choose values that reflect the accuracy of the measurement
+apparatus.
+
+The log-normal distribution was chosen to represent experimental metabolite and
+enzyme concentration measurements because the apparatuses used to measure these
+quantities typically have multiplicative errors. In other words, if the
+measured value is large, then the associated error is also proportionally
+large.
+
+
+Relative measurements
+---------------------
+
+In many realistic cases a measurement apparatus will give comparatively
+accurate information about the relative concentrations of some metabolites or
+enzymes while being comparatively uninformative as to their absolute
+values. While Maud currently does not support this kind of measurement, support
+is planned and will take the following form.
+
+
+Non-experimental Information
+============================
+
+Information that does not naturally take the form of an experimental
+measurement can be expressed in Maud's prior model. Maud allows users to
+specify independent log-normal priors for the following quantities:
+
+- enzyme :math:`kcat` parameters
+- enzyme/metabolite-in-compartment :math:`km` parameters
+- enzyme transfer constants
+- enzyme/metabolite-in-compartment inhibition and dissociation constants
+- enzyme concentrations
+- unbalanced metabolite concentrations
+
+For metabolite formation energies, which can be both negative and positive
+numbers, Maud allows users to specific independent normal priors.
+
+Users are encouraged to choose prior locations and scales using the method by
+calculating quantiles. Prior information is often easiest to ellicit in the
+form of qualitative statements like "it is very unlikely that :math:`kcat_e` is
+higher than 6.8 or lower than 0.4". Information in this form naturally
```

Comment:
It would be nice to have different shaped priors. For example Ben Goodrich's spline fitting? I'm not sure how that affects the gradient evaluation speed compared to the standard priors or if it's even feasible in Stan atm.

Reply:
Yeah it's definitely something worth considering - I think the lognormals probably tend to go a bit higher/lower than we want even when the quantiles are fixed.

I think we could definitely try implementing Ben Goodrich's idea, or maybe try using gamma distributions instead.